widget.json

```diff
@@ -21,12 +21,17 @@
       "vendor/bootstrap-colorpicker.css",
 
       "js/interface.templates.js",
+      "dist/core.js.map",
```

Comment:
why do you need map files listed? they won't make any different to the backend

Reply:
@squallstar I was having file not found error on iOS because of it

Reply:
@hcarneiro then just disable source maps from being generated in your gulpfile.

The dist files should not include this line: https://github.com/Fliplet/fliplet-widget-interactive-map/blob/master/dist/add-markers.js#L1567 

That's what is telling iOS to download the map file